security-whitepaper/cloud-native-security-whitepaper.md

```diff
@@ -0,0 +1,846 @@
+
+[https://owasp.org/www-community/Application_Threat_Modeling](https://owasp.org/www-community/Application_Threat_Modeling)
+
+[NIST Application Security Container Guide](https://nvlpubs.nist.gov/nistpubs/SpecialPublications/NIST.SP.800-190.pdf), [Center for Internet Security (CIS)](https://www.cisecurity.org/), [NIST Security Strategies for microservices](https://csrc.nist.gov/publications/detail/sp/800-204/final) and [OpenSCAP](https://www.open-scap.org/) benchmarks exist for [Docker](https://www.cisecurity.org/benchmark/docker/), [Kubernetes](https://www.cisecurity.org/benchmark/kubernetes/), and several managed Kubernetes distributions.
+
+[MITRE ATT&CK Matrix For Kubernetes](https://www.darkreading.com/threat-intelligence/microsofts-kubernetes-threat-matrix-heres-whats-missing/a/d-id/1339106)
+
+### Acknowledgements
+
+This white paper is a community effort driven by the members of the CNCF Security-SIG. Thank you to everyone for their outstanding contributions. Special thanks to Emily Fox and Jeyappragash JJ.
+
+Contributors:
+
+- Aradhna Chetal - [TIAA](http://tiaa.org/)
+- Brandon Lum - [IBM](https://www.ibm.com/)
+- Chase Pettet - [Mirantis](https://www.mirantis.com/) (Chase.Pettet@mirantis.com)
+- Emily Fox - [US National Security Agency (NSA)](https://www.nsa.gov/)
+- Gadi Naor - [Alcide](http://www.alcide.io/)
+- Harmeet Singh - [IBM](https://www.ibm.com/)
+- Jeff Lombardo - Independent
+- Jeyappragash JJ - [Tetrate IO](https://www.tetrate.io/)
+- Pushkar Joglekar - [Visa](https://visa.com/)
+- Rowan Baker - [ControlPlane](https://control-plane.io/)
+- Andrew Martin - [ControlPlane](https://control-plane.io/)
+- Trishank Karthik Kuppusamy - [Datadog](https://www.datadoghq.com/)
+- Vinay Venkataraghavan -[Prisma Cloud (Palo Alto Networks)](https://www.paloaltonetworks.com/prisma/cloud)
+- Wayne Haber - [GitLab](https://about.gitlab.com/)
+- Mark Bower
+- Alex Chircop - StorageOS
+
+Reviewers:
+
+- @justincappos
+- @lumjjb
+- @whaber
+- @craigbox
+- @anvega
+- @magnologan
+- Alok Raj - [XenonStack](https://www.xenonstack.com/) ([alok@xenonstack.com](mailto:alok@xenonstack.com))
+- @nyrahul
+- @ranio1
+
+[1](#sdfootnote1anc) Another model to consider is Cloud, Clusters, Containers, and Code: [https://kubernetes.io/docs/concepts/security/overview/](https://kubernetes.io/docs/concepts/security/overview/)
+
+[2](#sdfootnote2anc)[Shifting security left](https://www.devsecops.org/blog/2016/5/20/-security) often leaves organizations to lapse operational security monitoring. It is important that security exists in all parts of the lifecycle and organizations continually evaluate other aspects of their business and technology processes where they may reach beyond modern security paradigms to embrace security as a culture and habit.
+
+[3] Example - [MITRE ATT&CK Framework for Kubernetes](https://www.darkreading.com/threat-intelligence/microsofts-kubernetes-threat-matrix-heres-whats-missing/a/d-id/1339106)
```

Comment:
possible numbering issue because of duplicate 3

Reply:
The same footnote numbering is in the original document as well. @TheFoxAtWork can you please check?